environment.py

```diff
@@ -534,12 +534,25 @@ def env() -> Mapping[str, Optional[str]]:
         # develop branch, 0 otherwise. Personal deployments have this set to 0.
         #
         'AZUL_IS_SANDBOX': '0',
-        
+
         # A list of names of AWS IAM roles that should be given permission to
         # manage incidents with AWS support as defined in CIS rule 1.20:
         #
         # https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cis-controls.html#securityhub-cis-controls-1.20
         #
         #
-        'azul_aws_support_roles': json.dumps([])
+        'azul_aws_support_roles': json.dumps([]),
+
+        # Slack integration with AWS Chatbot app is disabled by default, enable
+        # it by replacing the None values. See section 3.1.3 of the README.
+        #
+        'slack_integration': json.dumps({
+            # The Slack workspace ID integrating with the AWS Chatbot app.
+            #
+            'workspace_id': None,
+
+            # The Slack channel ID receiving the AWS Chatbot app notifications.
+            #
+            'channel_id': None
+        })
```

Comment:
```suggestion
        'slack_integration': None
```

Instead, document the shape in the comment above.